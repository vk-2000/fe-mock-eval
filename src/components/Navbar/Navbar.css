nav {
    background-color: #38497E;
    height: 10vh;
    display: flex;
    justify-content: start;
    align-items: center;
}
.logo {
    font-size: x-large;
    margin: 0 2rem;
    color: white;
}
.logo span {
    font-weight: bold;
}