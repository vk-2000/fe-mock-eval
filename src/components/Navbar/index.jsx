import React from 'react';
import './Navbar.css';

const Navbar = () => (
  <nav>
    <div className="logo">
      My
      {' '}
      <span>Record</span>
      {' '}
      Shelf
    </div>
  </nav>
);

export default Navbar;
