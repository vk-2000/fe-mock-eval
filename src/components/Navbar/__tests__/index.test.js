import { render } from '@testing-library/react';
import Navbar from '..';

describe('Navbar', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Navbar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
