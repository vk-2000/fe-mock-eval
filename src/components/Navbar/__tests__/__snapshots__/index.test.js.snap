// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render correctly 1`] = `
<DocumentFragment>
  <nav>
    <div
      class="logo"
    >
      My 
      <span>
        Record
      </span>
       Shelf
    </div>
  </nav>
</DocumentFragment>
`;
